src/Widgets/SearchResultsWidget/SearchResultsWidgetComponent.tsx

```diff
@@ -98,9 +111,22 @@ provideComponent(SearchResultsWidget, ({ widget }) => {
   )
 })
 
-function TotalCountSummary({ totalCount }: { totalCount: number }) {
-  if (totalCount === 0) return 'No search results'
-  if (totalCount === 1) return '1 search result'
+const TotalCountSummary = connect(function TotalCountSummary({
+  totalCount,
+  widget,
+}: {
+  totalCount: number
+  widget: SearchResultsWidgetInstance
+}) {
+  const attributes = ['resultsHeadline0', 'resultsHeadline1'] as const
+  const attribute = attributes[totalCount] || 'resultsHeadline'
 
-  return `${totalCount} search results`
-}
+  return (
+    <ContentTag
+      tag="span"
+      content={widget}
+      attribute={attribute}
+      dataContext={{ count: totalCount.toString() }}
```

Comment:
@apepper @dcsaszar I just stumbled over this code 😨 

Passing hand-made "value objects" into `dataContext` is a deprecated, legacy API that was never released publicly. The SDK only keep it around because Neoletter still uses it. Please don't use it in the Portal. It's scheduled to be removed as soon as Neoletter no longer needs it.

Please remove this here as soon as possible, so that this API does not proliferate in any forks.

Reply:
Addressed in https://github.com/Scrivito/scrivito-portal-app/pull/506.